liquibase-integration-tests/src/test/resources/liquibase/extension/testing/command/calculateChecksum.test.groovy

```diff
@@ -40,7 +40,7 @@ Optional Args:
         ]
 
         expectedResults = [
-                statusCode   : 0
+                checksumResult   : "8:b6084e5d5f46b534bbbe18a0d35d34e0"
```

Comment:
This test will fail when Liquibase upgrades the checksum version. That failure indicates the test is doing its job and letting us know that a change occurred, which is good. Whatever statusCode is (exit code??) probably would not fail with a checksum version upgrade, so I like this enhancement!

Reply:
Validation of error messages added!